docs/_articles/en/user-guide/byotemplate.md

```diff
@@ -4,79 +4,40 @@ lang: en
-|Lightningapp   |DefaultLightningApp.app <br> DefaultLightningController.js <br> DefaultLightningCss.css <br> DefaultLightningHelper.js <br> DefaultLightningRenderer.js <br> DefaultLightningSVG.svg <br> DefaultLightningAuradoc.auradoc <br>auradefinitionbundle.app-meta.xml
-|Lightningcomponent |aura/default/default.cmp <br> aura/default/default.cmp-meta.xml <br> aura/default/defaultController.js <br> aura/default/defaultHelper.js <br> aura/default/defaultRenderer.js <br> aura/default/default.svg <br> aura/default/default.design <br> aura/default/default.css <br> aura/default/default.auradoc <br> lwc/default/default.js<br> lwc/default/default.html <br> lwc/default/default.js-meta.xml |
-|Lightningevent | defaultLightningEvt.evt <br> auradefinitionbundle.evt-meta.xml
-|Lightninginterface | DefaultLightningIntf.intf <br> auradefinitionbundle.intf-meta.xml
-|Lightningtest | DefaultLightningTest.resource
-|Project | Any file in the project folder
-|Staticresource | empty.resource <br> empty.js <br> empty.json <br> empty.css <br> empty.txt <br> _staticresource.resource-meta.xml
-|visualforcecomponent| DefaultVFComponent.component <br> _component.component-meta.xml
-|visualforcepage | DefaultVFPage.page <br> _page.page-meta.xml
-
-## Clone the Repo
-See GitHub documentation for instructions on how to clone repos.
+**Note**:
+This [git repo subdirectory](https://github.com/forcedotcom/salesforcedx-templates/tree/main/src/templates) contains a collection of official Salesforce templates for metadata components. Only updates made to the files listed in this directory show up in source files in VS Code. You can clone this subdirectory, or store your custom templates locally. Keeping the same folder structure, just update relevant template files with your code. Remove the files that you don’t wish to override. The Salesforce Extensions require your templates to follow the exact folder structure and nomenclature as this repo. For example, your Apex class templates must be in a folder named ``apexclass`` in a file named ``DefaultApexClass.cls``. There’s no such restriction when you use the CLI to specify templates for metadata objects. You do not need the rest of the repo `salesforcedx-templates`.
 
 ## Set Default Template Location
-1. Open the `sfdx-config.json` config file in your `<project-folder>/.sfdx` folder
-2. Add the parameter `customOrgMetadataTemplates` and set its value to the templates folder in your cloned repo. For example:
-```
-{
-  "customOrgMetadataTemplates": "https://github.com/mygitrepo/salesforcedx-templates/tree/main/packages/templates/src/templates"
-}
-```
-
-or the location of a local copy:
+ You can store template files in a local directory, or in a github project. Set ``customOrgMetadataTemplates`` to your custom template location:
+1. Run ``sfdx config:set customOrgMetadataTemplates=<github repo or local template directory>`` command from the terminal inside VS Code. 
```

Comment:
Ah ha - some of the comments in the note now make sense but I still would prefer to see that info in context.